src/Compilers/CSharp/Test/Emit/Emit/IsConstModifierTests.cs

```diff
@@ -0,0 +1,757 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Emit
+{
+    public class IsConstModifierTests : CSharpTestBase
```

Comment:
Similarly we should have tests for explicit interface implementations against interface methods that use ref readonly and have/don't have IsConst modifiers.


---
In reply to: [118076522](https://github.com/dotnet/roslyn/pull/19658#discussion_r118076522) [](ancestors = 118076522)